### ISSUE TITLE
Query-batching doesn't work

### ISSUE THREAD

**Issue by green2k** [APIs-guru/graphql-faker#19]
Hi, does this graphql-faker "middleware" support query-batching? 

I always get an error that the input JSON is invalid :disappointed:

**Comment by IvanGoncharov** [comment]
@green2k Do you mean query-batching implemented by Apollo server?

**Comment by green2k** [comment]
@IvanGoncharov Yup -> http://dev.apollodata.com/core/network.html#query-batching

**Comment by IvanGoncharov** [comment]
@green2k It vendor extension and `express-graphql` (reference server) which we use in Faker doesn't support it.
Not sure how to add support for it. I have an idea of exporting Faker as express middleware so you can use it with Apollo Server but it will require a huge time investment.

Meanwhile, I need to add explicit batching detection and throw a better error message.

How critical is batching support for your use cases? Is it easy to disable batching on client?

**Comment by lomereiter** [comment]
This would be nice to have, but disabling batching on the client is just too easy to put any effort here. An error message would be good, though, as it's not immediately obvious what's wrong.
